config/initializers/devise.rb

```diff
@@ -216,14 +216,14 @@
   # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
   # :both  = Enables both strategies
   # :none  = No unlock strategy. You should handle unlocking by yourself.
-  # config.unlock_strategy = :both
+  config.unlock_strategy = :both
 
   # Number of authentication tries before locking an account if lock_strategy
   # is failed attempts.
-  # config.maximum_attempts = 20
+  config.maximum_attempts = 3
```

Comment:
20 attempts seemed high. Let me know if you think 3 is too low.

Reply:
Personal but not strong opinion: I know that it's common for banks to use 3 but I feel like we can be a bit more lenient and maybe use 5.

Reply:
Sure, I had 5 at first, then lowered it for local testing, happy to increase it again. Would like to see what others think.

Reply:
I like 5, 3 seems too low to me as well.